pkg/acsengine/testdata/key-vault-params/kubernetes_expected.json

```diff
@@ -675,9 +675,25 @@
         "description": "The azure deploy environment. Currently support: AzureCloud, AzureChinaCloud"
       },
       "type": "string"
+    },
+    "windowsAdminPassword": {
+      "metadata": {
+        "description": "Password for the Windows Swarm Agent Virtual Machines."
```

Comment:
Not related to this commit, this is a typo in windowsparams.t and we can fix in another issue/PR.